godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -79,7 +81,8 @@ class TractViewController: BaseViewController {
 
     override func displayScreenTitle() {
         if parallelLanguageIsAvailable() && determinePrimaryLabel() != determineParallelLabel() {
-            self.navigationItem.titleView = languageSegmentedControl()
+            configureLanguageSegmentedControl()
+            self.navigationItem.titleView = languageSegmentedControl
```

Comment:
No, it's so that I can easily access the segmented control in the delegate method. I would have to go through the navigation controller to the navigation bar and to the title view, and then get its subview and cast it so a segmented control to check the value of it. Far easier to just hold a reference :)